plugins/modules/omevv_firmware.py

```diff
@@ -953,9 +937,9 @@ def is_update_job_allowed(self, vcenter_uuid, cluster_group_id, cluster_name):
         Returns:
             bool: True if the update job is allowed, False otherwise.
         """
-        try:
```

Comment:
Revert the changes, as we have fixed defect for failed as true.

Reply:
Incorporated the comment